app/scripts/lib/account-tracker.test.js

```diff
@@ -908,75 +829,4 @@ describe('Account Tracker', () => {
       });
     });
   });
-
-  describe('_updateAccountsViaBalanceChecker', () => {
```

Comment:
Same question here — do the tests elsewhere cover the behavior that was previously tested here?

Reply:
yup! covered in updateAccount now. Different entrypoint, but equivalent testing